x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/effect_scope.tsx

```diff
@@ -121,6 +132,7 @@ export const WithContextMenu = memo<WithContextMenuProps>(
             {children}
           </EuiButtonEmpty>
         }
+        disabled={!canReadPolicies}
```

Comment:
I don't think its a good idea to display this with a disabled Button. When the user is not allowed to Read policies, then we should just display the policy name using regular text (`<EuiText />`). It looks strange (to me) for those entries to be displayed grey'ed out in the UI.

Thoughts?

Reply:
I agree, so with some more effort, here is a new version: babbcf2

(I've added screenshots to the description)